dataactvalidator/scripts/loaderUtils.py

```diff
@@ -5,12 +5,12 @@
 def clean_col_names(field):
     """Define some data-munging functions that can be applied to pandas
     dataframes as necessary"""
-    return str(field).lower().strip().replace(" ","_").replace(",","_")
+    return str(field).lower().strip().replace(" ", "_").replace(",", "_")
 
 
 class LoaderUtils:
     @classmethod
-    def padFunction(self, field, padTo, keepNull):
+    def padFunction(cls, field, padTo, keepNull):
```

Comment:
Yeah, I was going to do all the moving of functions and fixing function names (under_score instead of camelCase) in the next round :)